etc/gedit.profile

```diff
@@ -18,10 +16,14 @@ include /etc/firejail/disable-programs.inc
 
 include /etc/firejail/whitelist-var-common.inc
 
+# following line makes settings immutable
+apparmor
 caps.drop all
-# net none - makes settings immutable
 machine-id
+net none
```

Comment:
Should we comment out `apparmor` and `net none` to avoid immutable settings?

Reply:
Oops! I overlooked `net none` when I claimed that everything is reverted to the situation before. Sorry!

To summarize: `net none` will continue to affect Mint 18, Ubuntu 16.04, CentOS... I guess that's about it. Debian Jessie will reach EOL soon. With Mint 19 and the next Ubuntu LTS waiting, I would find it defensible to enable `net none` by default in these profiles, but we can also comment it again, then truly reverting to the original situation.

Regarding apparmor I personally would tend to a yes, also because, as @Vincent43 has pointed out in another thread, patches are finally in the pipeline to bring D-Bus mediation into the mainline kernel. Any other opinions regarding apparmor?

Reply:
So do I understand it right that consensus is emerging to disable all of `apparmor`, `nodbus`, `net none` when they freeze application settings?

(in case yes, how about an exception for the archive managers file-roller and engrampa? Because they have hardly any setting to change....)

Reply:
I think we should do that with this release and then re-think our approach afterwards.
Let's do that exception for the archive managers!